sdk/confidentialledger/azure-confidentialledger/README.md

```diff
@@ -353,6 +363,71 @@ ledger_client = ConfidentialLedgerClient(
 )
 ```
 
+### Verify write transaction receipts
+
+Clients can leverage the receipt verification library in the SDK to verify write transaction receipts issued by Azure Confidential Legder instances. The utility can be used to fully verify receipts offline as the verification algorithm does not require to be connected to a Confidential ledger or any other Azure service.
+
+Once a new entry has been appended to the ledger (please refer to [this example](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/confidentialledger/azure-confidentialledger#append-entry)), it is possible to get a receipt for the committed write transaction.
+
+```python
+from azure.confidentialledger import ConfidentialLedgerClient
+from azure.confidentialledger.certificate import ConfidentialLedgerCertificateClient
+from azure.identity import DefaultAzureCredential
+
+# Replace this with the Confidential Ledger ID 
+ledger_id = "my-ledger-id"
+
+# Setup authentication
+credential = DefaultAzureCredential()
+
+# Create a Ledger Certificate client and use it to
+# retrieve the service identity for our ledger
+identity_client = ConfidentialLedgerCertificateClient()
+network_identity = identity_client.get_ledger_identity(
+    ledger_id=ledger_id
+)
+
+# Save ledger service certificate into a file for later use
+ledger_tls_cert_file_name = "ledger_certificate.pem"
+with open(ledger_tls_cert_file_name, "w") as cert_file:
+    cert_file.write(network_identity["ledgerTlsCertificate"])
+
+# Create Confidential Ledger client
+ledger_client = ConfidentialLedgerClient(
+    endpoint=f"https://{ledger_id}.confidential-ledger.azure.com",
+    credential=credential,
+    ledger_certificate_path=ledger_tls_cert_file_name
+)
+
+# The method begin_get_receipt returns a poller that
+# we can use to wait for the receipt to be available for retrieval 
+get_receipt_poller = ledger_client.begin_get_receipt(transaction_id)
+get_receipt_result = get_receipt_poller.result()
+
+print(f"Write receipt for transaction id {transaction_id} was successfully retrieved: {get_receipt_result}")
+```
+
+After fetching a receipt for a write transaction, it is possible to call the `verify_receipt` function to verify that the receipt is valid.
+
+```python
+from azure.confidentialledger.receipt import (
+    verify_receipt,
+)
+
+# Read contents of service certificate file saved in previous step.
+with open(ledger_tls_cert_file_name, "r") as service_cert_file:
+    service_cert_content = service_cert_file.read()
```

Comment:
If we think reading this data from a file will be a common scenario - we could add support for `IO` as an input directly into `verify_receipt`. Not important now - can be an additive change in future.

Reply:
Not a common scenario I would say: the receipt verification library will be mostly used in conjunction with the `GetReceipt` API, so customers would use it directly in the same format they received it from the API. We could support reading from file in the future if we believe there are enough customer use cases for this, though.